bravado/mapping/request.py

```diff
@@ -0,0 +1,60 @@
+from bravado.mapping.operation import log
+from bravado.mapping.param import unmarshal_param
+
+
+class RequestLike(object):
+    """
+    Define a common interface for bravado to interface with server side
+    request objects.
+
+    Subclasses are responsible for providing attrs for __required_attrs__.
+    """
+    __required_attrs__ = [
+        'path',     # dict of URL path parameters
+        'params',   # dict of parameters from the query string and request body.
```

Comment:
http://docs.pylonsproject.org/docs/pyramid/en/latest/api/request.html#pyramid.request.Request.GET

`request.GET` has all query params, regardless of http method